packages/suite/src/utils/wallet/coinjoinUtils.ts

```diff
@@ -134,12 +136,26 @@ export const transformFeeRatesMedians = (medians: CoinjoinStatusEvent['feeRatesM
  * - coordinatorFeeRate: multiply the amount registered for coinjoin by this value to get the total fee
  * - feeRatesMedians: array => object with values in kvBytes
  */
-export const transformCoinjoinStatus = (event: CoinjoinStatusEvent) => ({
-    rounds: event.rounds.map(r => ({ id: r.id, phase: r.phase })),
-    coordinationFeeRate: event.coordinationFeeRate,
-    feeRatesMedians: transformFeeRatesMedians(event.feeRatesMedians),
-    allowedInputAmounts: event.allowedInputAmounts,
-});
+export const transformCoinjoinStatus = ({
+    feeRatesMedians,
+    allowedInputAmounts,
+    rounds,
+    coordinationFeeRate,
+}: CoinjoinStatusEvent) => {
+    const feeRatesMediansTransformed = transformFeeRatesMedians(feeRatesMedians);
+    const txSize = getInputSize('Taproot') + getOutputSize('Taproot');
+    const minWithFee = allowedInputAmounts.min + txSize * feeRatesMediansTransformed.recommended;
```

Comment:
i wondering, wouldnt be better to calculate it directly in coinjoin package?
we already do some transformation there, see src/client/Status.ts processStatus method

Reply:
I tried it there first, but I'd rather move the fee rates medians transformation there too. Is that ok? Maybe even rounds mapping?

Reply:
Coming back to this, I'm thinking that since it is a derived value, `minWithFee` should not be in the redux state at all and could just calculated in selector 🤔

Reply:
> but I'd rather move the fee rates medians transformation there too. Is that ok?

i guess its ok, its always better to have all transformations in one place

Reply:
Agreed this is a redundancy, you could either replace the `min` value with this or make the calculation in the selector. Also, make sure to use the correct value to set `amountTooSmallForCoinjoin` in `UtxoSelection`.

Reply:
So I moved the transformations to client/Status.ts 214e7a591769c6f737bd9b2bc93cc341a09347b8, and in reducer there is only removal of `changed` and selecting only id and phase for each round to be stored. 

minWithFee is derived in selector 73e05fdb6f73c23f3fe95be09703022ae612d7e9

`min` and `minWithFee` have different meaning. Even utxo with value between min and minWithFee can be selected for coinjoin if you have other utxos to cover the fee. See https://tbtc2.trezor.io/tx/f9a995c914d8213e6923064a569eb3b8488d40fbe261d544d6135b05f57c3482 
It can be uneconomical for you, but thats another issue https://github.com/trezor/trezor-suite/issues/7377